ll.c

```diff
@@ -432,6 +432,25 @@ void stfs_ll_op_statfs(fuse_req_t req, fuse_ino_t ino) {
 	}
 }
 
+void sqfs_ll_op_init(void *userdata, struct fuse_conn_info *conn) {
+	sqfs_ll *ll = userdata;
+
+	if (!ll->fs.notify_pipe) {
+		return;
```

Comment:
It's easy to miss that this will do the wrong thing if we ever add more logic to sqfs_ll_op_init. Maybe we can just make our common function (see below) return early if notify_pipe is null?